tests/main/reference/hvdc_HvdcPQPropDangling/outputIIDM.xml

```diff
@@ -3,35 +3,35 @@
     <iidm:substation id="_BUS____1_SS" name="BUS    1_SS" country="AF">
         <iidm:voltageLevel id="_BUS____1_VL" name="BUS    1_VL" nominalV="400" topologyKind="BUS_BREAKER">
             <iidm:busBreakerTopology>
-                <iidm:bus id="_BUS____1_TN" v="400.01716256313972" angle="0.014619844494811761"/>
+                <iidm:bus id="_BUS____1_TN" v="400.01131969356123" angle="0.01463373467219858"/>
             </iidm:busBreakerTopology>
-            <iidm:generator id="_Generator___1" name="GEN    1" energySource="OTHER" minP="-9999" maxP="9999" voltageRegulatorOn="true" targetP="1000" targetV="400" targetQ="0" bus="_BUS____1_TN" connectableBus="_BUS____1_TN" p="-800.0749821623865" q="165.18577341237216">
+            <iidm:generator id="_Generator___1" name="GEN    1" energySource="OTHER" minP="-9999" maxP="9999" voltageRegulatorOn="true" targetP="1000" targetV="400" targetQ="0" bus="_BUS____1_TN" connectableBus="_BUS____1_TN" p="-800.07492196163207" q="168.94890995994828">
```

Comment:
it changes a lot the results of the computation 65 -> 168 for example but it is ok I guess ?

Reply:
yes this is expected